client/go/commands/usage_templates.go

```diff
@@ -92,3 +92,59 @@ Example:
     heketi -server http://localhost:8080 cluster list
 
 `
+var usageTemplateNode = `Node is a command used for managing Heketi nodes.
+
+Usage:
+    heketi -server [server] [options] node [subcommand]
+
+The subcommands are:
+    add        Adds a new node to the designated cluster.
+    info       Returns information about the specific node.
+    destroy    Destroys node with specified id.
+
+Use "heketi node [subcommand] -help" for more information about a subcommand
+
+`
+var usageTemplateNodeAdd = `node add is a command used for adding a node to the specified cluster.
+
+Usage:
+    heketi -server [server] [options] node add [flags]
+
+The flags are:
+    -zone                  The zone in which the node should reside.
```

Comment:
I'm not sure if we need to do the flags printout... I think we need to look into the flags API before we continue this model.  I have used the flags API and it prints out the string given when defining the flag.  Maybe we are not setting it up correctly, like in `cmd.flags.Usage = ....`